server/service/src/ledger_fix/fixes/adjust_invoice_status.rs

```diff
@@ -0,0 +1,376 @@
+use repository::{
+    activity_log::{
+        ActivityLogFilter, ActivityLogRepository, ActivityLogSort, ActivityLogSortField,
+    },
+    stock_line_ledger::{StockLineLedgerFilter, StockLineLedgerRepository},
+    ActivityLogRow, ActivityLogType, EqualFilter, InvoiceLineFilter, InvoiceLineRepository,
+    InvoiceRow, InvoiceRowRepository, InvoiceStatus, Pagination, StockLineRowRepository,
+    StorageConnection,
+};
+
+use crate::ledger_fix::{fixes::LedgerFixError, ledger_balance_summary, LedgerBalanceSummary};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Ord, PartialOrd)] // Wanted to use InvoiceStatus, but need a good "other" variant
```

Comment:
Note derive `Ord` and `PartialOrd` here means the order of the variants in the enum definition here is important. Without extensive unit tests it'd be pretty easy to break if innocently adding a status...